src/main/kotlin/inc/evil/d17map/controllers/ReservationController.kt

```diff
@@ -20,6 +20,11 @@ import java.util.*
 @RequestMapping("/reservations")
```

Comment:
A to tylko tu pamiętaj, żeby to usunąć, bo teraz to mamy "/reservations/buildings/{id}/reservations"